app/src/main/java/com/todoist_android/ui/Utils.kt

```diff
@@ -20,10 +20,8 @@ import java.text.SimpleDateFormat
 import java.util.*
 
 
-const val BASE_URL = "https://621ce943768a4e1020b93731.mockapi.io/api/v1/"
-//const val BASE_URL = "http://192.168.100.224:8080/" //when running backend locally, use your laptop ip address
-
-//const val BASE_URL ="http://192.168.0.108:8080/api"
+//const val BASE_URL = "https://621ce943768a4e1020b93731.mockapi.io/api/v1/"
+const val BASE_URL = "http://192.168.100.224:8080/" //when running backend locally, use your laptop local ip address
```

Comment:
@Njumbi remember to always change to your laptop's local ip address

Reply:
okay

Reply:
Also since Backend is working on login with email address (current is username), so you can comment out line 110 - 115 of LoginFragment.kt to be able to successfully log in @Njumbi